spec/SPEC.md

```diff
@@ -241,59 +241,104 @@ or
 gitoid:blob:sha256\n
 ```
 
-All identifiers in a OmniBOR Document MUST be of the Artifact Identifier
+All identifiers in a Input Manifest MUST be of the Artifact Identifier
 Type declared in the header.
 
-#### OmniBOR Document Child Records
+#### Input Manifest Records
 
-The OmniBOR Document after the header consists of a list of newline terminated
-child records
+The Input Manifest after the header consists of a list of newline terminated
+input records
 
-A child artifact which is itself a leaf artifacts would be represented by:
+An input record for an artifact for which no Input Manifest Identifier is known is represented as:
 
 ```
-blob⎵${artifact id of child}\n
+blob⎵${artifact identifier of the input artifact}\n
 ```
 
-A child artifact which is itself a derived artifact would be represented by
+An input record for an artifact for which an Input Manifest Identifier is known is represented as:
 
 ```
-blob⎵${artifact id of child}⎵bom⎵${OmniBOR identifier of child's OmniBOR Document}\n
+blob⎵${artifact identifier of the input artifact}⎵bom⎵${input manifest identifier of the input artifact}\n
 ```
 
-The child artifact records must be written to the OmniBOR Document in lexical
+```⎵``` above refers to the ascii space character (0x20).
```

Comment:
"ascii" -> "ASCII"

Reply:
Fixed in https://github.com/omnibor/spec/pull/51/commits/9e9cf01385226c565176b6a46802dfe8d0df9c46